askama_actix/Cargo.toml

```diff
@@ -14,7 +14,7 @@ edition = "2018"
 
 [dependencies]
 actix-web = { version = "4", default-features = false }
-askama = { version = "0.11.2", path = "../askama", default-features = false, features = ["with-actix-web"] }
+askama = { version = "0.11", path = "../askama", default-features = false, features = ["with-actix-web"] }
```

Comment:
generally i thought due to the new resolver that it would auto pull in the latest patch version of askama. but i guess i should add this back just in case they might be using the old resolver.